components/Chips/examples/ChipsFieldDeleteEnabledViewController.swift

```diff
@@ -15,14 +15,22 @@
 import UIKit
 
 import MaterialComponents.MaterialChips
-import MaterialComponents.MaterialChips_ChipThemer
 import MaterialComponents.MaterialTextFields
+import MaterialComponentsBeta.MaterialChips_Theming
+import MaterialComponentsBeta.MaterialContainerScheme
 
 class ChipsFieldDeleteEnabledViewController : UIViewController, MDCChipFieldDelegate {
   var colorScheme = MDCSemanticColorScheme()
   var typographyScheme = MDCTypographyScheme()
   var chipField = MDCChipField()
 
+  var scheme: MDCContainerScheming {
```

Comment:
Would you please add the shape scheme either here or as a follow-up?

Reply:
done